curriculum/challenges/english/07-scientific-computing-with-python/learn-classes-and-objects-by-building-a-sudoku-solver/6606e2f27f19ca5f398c6aed.md

```diff
@@ -0,0 +1,83 @@
+---
+id: 6606e2f27f19ca5f398c6aed
+title: Step 52
+challengeType: 20
+dashedName: step-52
+---
+
+# --description--
+
+Next, you'll work on a method that attempts to solve the sudoku in-place, meaning it would modify the existing sudoku board rather than creating a new one.
```

Comment:
```suggestion
Next, you'll work on a method that attempts to solve the sudoku in-place, meaning it will modify the existing sudoku board rather than creating a new one.
```